agent/cmd/cuda-vmm-interpose/interpose.c

```
@@ -0,0 +1,1779 @@
+    pthread_mutex_unlock(&state_lock);
+    return CUDA_ERROR_NOT_READY;
+  }
+  result = function != NULL ? function(address, size) : unavailable();
+  if (result == CUDA_SUCCESS)
+    mapping->mapped = false;
+  pthread_mutex_unlock(&state_lock);
+  return result;
+}
+
+CUresult CUDAAPI
+cuMemSetAccess(CUdeviceptr address, size_t size, const CUmemAccessDesc* descriptors, size_t count)
```

Cppcheck (2.21.0)

[style] The function 'cuMemSetAccess' should have static linkage since it is not used outside of its translation unit.

(staticFunction)